### DISCUSSION TITLE
High CPU Load

### DISCUSSION THREAD

**Discussion by AMWMusic** [apohl79/audiogridder#1275] [Q&A]
So far I have made the experience that version 1.2.0 seem to have a much higher CPU load than version 1.1.1, so I had to go back to 1.1.1 to have projects running, that don't run anymore with 1.2.0. This is strange in so far, that the buffer in the new version can be set to higher values. Is it possible to configure version 1.2.0 in a way that it works like 1.1.1?

**Comment by apohl79** [comment]
Do you generally see a higher load or with specific plugins?

**Reply by AMWMusic** [reply]
Thank you for your reply! I rather would say, this occurs in general. Version 1.1.1 has no monitor for individual plugins, as far as I am aware of. But in 1.2.0 the CPU load increases significantly sometimes with regard to many plugins. It can also occur, that the load decreases and suddently increases again. As a global phenomenon working with version 1.2.0 is nearly impossible, because too many audio breaks are occurring even with maximum buffer, that don't occur in the same way with version 1.1.1, although there is also a point, when they occur in this version.

**Comment by AMWMusic** [comment]
I have installed two new network cards, but this does't help much unfortunately regarding the issues with AG 1.2.0. The ms values and FX load is higher, as it seems, regarding the same project. For example, if the FX load is in AA 1.1.1 about 70%, the same project in AA 1.2.0 can be about 100%, that proper monitoring isn't possible anymore.
